x2i-data/apta.yaml

```diff
@@ -0,0 +1,993 @@
+  - - +?
+    - ̚
+  - - +A
+    - ̂
+  - - +B
+    - ᷇
+  - - +C
+    - ᷈
+  - - +D
+    - ᷉
+  - - +N
+    - ͊
+  - - +O
+    - ͗
+  - - +R
+    - ˞
+  - - +V
+    - ͌
+  - - +^
+    - ̑
+  - - +c
+    - ͑
+  - - +d
+    - ͊
+  - - +u
+    - ̆
+  - - +x
+    - ̽
+  - - +{
+    - ͆
+  - - +}
+    - ͡
+  - - -~
+    - ̰
+  - raw:
+      - '\+?~'
+      - ̃
+  - raw:
+      - '\-["”]'
+      - ̤
+  - - "-+"
+    - ̟
+  - - "--"
+    - ̠
+  - - "-."
+    - ̣
+  - - "-0"
+    - ̥
+  - - "-<"
+    - ͔
+  - - "-="
+    - ̩
+  - - "->"
+    - ͕
+  - - "-?"
+    - ͉
+  - - "-A"
+    - ͇
+  - - "-L"
+    - ̴
+  - - "-O"
+    - ̹
+  - - "-P"
+    - ͎
+  - - "-R"
+    - ̢
+  - - "-S"
+    - ͍
+  - - "-T"
+    - ̞
+  - - "-^"
+    - ̯
+  - - "-a"
+    - ̺
+  - - "-c"
+    - ̜
+  - - "-j"
+    - ̡
+  - - "-l"
+    - ̻
+  - - "-m"
+    - ̼
+  - - "-p"
+    - ̙
+  - - "-q"
+    - ̘
+  - - "-r"
+    - ̝
+  - - "-s"
+    - ͈
+  - - "-t"
+    - ̞
+  - - "-v"
+    - ̬
+  - - "-w"
+    - ̫
+  - - "-{"
+    - ̪
+  - - "-}"
+    - ͜
+  - - "#'"
+    - ˚
+  - - "#+"
+    - ˖
+  - - "#,"
+    - ˳
+  - - "#-"
+    - ˗
+  - - "#0"
+    - ∅
+  - delimiters: ['#(', ')']
+    translations:
+      - - "1"
+        - ˩
+      - - "2"
+        - ˨
+      - - "3"
+        - ˧
+      - - "4"
+        - ˦
+      - - "5"
+        - ˥
+  - - "#1"
+    - ˩
+  - - "#2"
+    - ˨
+  - - "#3"
+    - ˧
+  - - "#4"
+    - ˦
+  - - "#5"
+    - ˥
+  - - "#6"
+    - ˩˧
+  - - "#7"
+    - ˩˥
+  - - "#8"
+    - ˧˥
+  - - "#9"
+    - ˧˩
+  - - "#<"
+    - ←
+  - - "#="
+    - ‿
+  - - "#>"
+    - →
+  - - "#?"
+    - ˹
+  - - "#A"
+    - ˥˩
+  - - "#B"
+    - ˥˧
+  - - "#C"
+    - ˩˥˩
+  - - "#D"
+    - ˥˩˥
+  - - "#E"
+    - ↑
+  - - "#F"
+    - ↘
+  - - "#I"
+    - ↓
+  - - "#O"
+    - ˒
+  - - "#R"
+    - ↗
+  - - "#T"
+    - ˕
+  - - "#c"
+    - ˓
+  - - "#o"
+    - "◌"
+  - - "#p"
+    - ꭫
+  - - "#q"
+    - ꭪
+  - - "#r"
+    - ˔
+  - - "#t"
+    - ˕
+  - raw: 
+      - '\#?\|'
+      - '|'
+  - - "!"
+    - ǃ
+  - raw: 
+      - '["”]'
+      - '’'
+  - - $
+    - ɬ
+  - - "%"
+    - ħ
+  - - "&"
+    - æ
+  - - "'"
+    - ˈ
+  - - ","
+    - ˌ
+  - - "0"
+    - ø
+  - - "1"
+    - ɨ
+  - - "2"
+    - ʌ
+  - - "3"
+    - ɜ
+  - - "4"
+    - ɾ
+  - - "5"
+    - ʎ
+  - - "6"
+    - ɐ
+  - - "7"
+    - ɤ
+  - - "8"
+    - ɒ
+  - - "9"
+    - œ
+  - - ":"
+    - ː
+  - - =
+    - ̩
+  - - ">"
+    - ̥
+  - - "?"
+    - ʔ
+  - - "@"
+    - ə
+  - - A
+    - ɑ
+  - - B
+    - β
+  - - C
+    - ç
+  - - D
+    - ð
+  - - E
+    - ɛ
+  - - F
+    - ɟ
+  - - G
+    - ɣ
+  - - H
+    - ɦ
+  - - I
+    - ɪ
+  - - J
+    - ɲ
+  - - K
+    - ʁ
+  - - L
+    - ɫ
+  - - M
+    - ɯ
+  - - N
+    - ŋ
+  - - O
+    - ɔ
+  - - P
+    - ɸ
+  - - Q
+    - ʕ
+  - - R
+    - ɹ
+  - - S
+    - ʃ
+  - - T
+    - θ
+  - - U
+    - ʊ
+  - - V
+    - ʋ
+  - - W
+    - ʉ
+  - - X
+    - χ
+  - - Y
+    - ʏ
+  - - Z
+    - ʒ
+  - - "{"
+    - ̪
+  - - "}"
+    - ͡
+  - raw:
+      - '(.)([͜͢͡])'
+      - $2$1
```

Comment:
This rule isn't correct because of combining diacritics. There might be some `\P{property}` syntax you can use to make it skip combining diacritics -- I use XRegExp, not plain ECMAScript RegExp -- but this is why I changed Z-SAMPA to use `t)s` instead of `ts)`.